### PR TITLE
fix(issuer): modify investment tier

### PR DESCRIPTION
**Pull request by sajclarke**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [ ] I linked the issues related to this PR in its description (if any).
- [ ] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

### CODE REVIEW

File: packages/polymath-ui/src/components/inputs/NumberInput/index.js [hunk 1]
```diff
@@ -66,7 +66,7 @@ export class NumberInput extends Component<Props, State> {
     max: Infinity,
     maxDecimals: 18, // max amount of decimals supported by solidity
     useBigNumbers: false,
-    value: null,
+    value: '',
```
Comment (Grsmto) [inline packages/polymath-ui/src/components/inputs/NumberInput/index.js]: @RafaelVidaurre you might want to take a look at this.

Comment (sajclarke) [reply packages/polymath-ui/src/components/inputs/NumberInput/index.js]: Thanks for pointing that out @Grsmto . I left that change by accident

File: packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js [hunk 2]
```diff
@@ -264,14 +289,20 @@ class InvestmentTiers extends React.Component<Props, State> {
...
         <FieldArray
           name="investmentTiers.tiers"
-          render={({ push }) => (
+          render={({ push, replace }) => (
             <Field
               name="investmentTiers.newTier"
               ticker={ticker}
               component={AddTierModal}
-              title={`Add the Investment Tier #${value.tiers.length + 1}`}
+              title={
```
Comment (RafaelVidaurre) [inline packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js]: Could we move this render logic inside the AddTierModal? (Which should now be renamed to `TierModal` or `TierFormModal` or something similar since it also edits now

File: packages/polymath-ui/src/components/inputs/NumberInput/index.js [hunk 3]
```diff
@@ -84,7 +84,7 @@ export class NumberInput extends Component<Props, State> {
 
   static getDerivedStateFromProps(props: Props, state: State) {
     const { oldValue } = state;
-    const { value, useBigNumbers, min, max, name } = props;
+    let { value, useBigNumbers, min, max, name } = props;
```
Comment (RafaelVidaurre) [inline packages/polymath-ui/src/components/inputs/NumberInput/index.js]: Why was this changed to `let`?

File: packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js (hunk 2 above)
Comment (sajclarke) [reply packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js]: Resolved with latest commit

File: packages/polymath-ui/src/components/inputs/NumberInput/index.js (hunk 3 above)
Comment (sajclarke) [reply packages/polymath-ui/src/components/inputs/NumberInput/index.js]: This was a mistake. Reverted with latest commit

**Comment by RafaelVidaurre** [comment]
Looks good to me, let's get @Grsmto 's green-light for this one

File: packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js [hunk 4]
```diff
@@ -109,11 +113,15 @@ class InvestmentTiers extends React.Component<Props, State> {
...
   };
 
-  handleAddNewTier = () => {
-    this.setState({ isAddingTier: true });
+  handleTierModal = (id, data) => {
+    if (typeof id === 'number') {
+      this.setState({ tierData: { id, ...data }, isAddingTier: true });
+    } else {
+      this.setState({ isAddingTier: true });
+    }
```
[the remark this answers is not in the document]
Comment (sajclarke) [reply packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js]: Thanks @Grsmto and @RafaelVidaurre I've applied the changes as requested
